sale_invoice_blocking/models/invoice_blocking_reason.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class InvoiceBlockingReason(models.Model):
+    _name = "invoice.blocking.reason"
+    _description = "Sale invoice blocking reason"
+
+    name = fields.Char(string="Motif")
```

Comment:
```suggestion
    name = fields.Char(string="Motif", required=True)
```
And maybe add a unique SQL constrain?